@shared/api/internal.ts

```diff
@@ -280,14 +279,32 @@ export const getAccountBalances = async ({
     networkPassphrase,
   });
 
-  let balances = null;
+  let balances: any = null;
   let isFunded = null;
   let subentryCount = 0;
 
   try {
-    ({ balances, subentryCount } = await dataProvider.fetchAccountDetails());
-    // let's filter out liquidity pool shares until freighter and wallet-sdk support
-    balances = omitBy(balances, (b: any) => b.liquidity_pool_id) as Balances;
+    const resp = await dataProvider.fetchAccountDetails();
+    balances = resp.balances;
+    subentryCount = resp.subentryCount;
+
+    for (let i = 0; i < Object.keys(resp.balances).length; i++) {
+      const k = Object.keys(resp.balances)[i];
+      const v: any = resp.balances[k];
```

Comment:
this is totally fine - but just thought I'd mention you could use `Object.entries` to get a tuple of the key and the value so you wouldn't have to declare `v` manually here

Reply:
ya  I couldn't figure out a way to cast `v` to type `any` in the for loop 🤔 . I wish something like `for (let [k, v: any] of Object.entries(resp.balances))` works but typescript doesn't like it 😢 

I'll merge but if there's a way to do it better I can change in next PR